WordPress/Docs/DB/RestrictedFunctionsStandard.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0"?>
+<documentation xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="https://phpcsstandards.github.io/PHPCSDevTools/phpcsdocs.xsd"
+    title="Restricted Database Functions"
+    >
+    <standard>
+    <![CDATA[
+      Avoid touching the database directly with PHP library functions. Use the $wpdb object and associated functions instead.
```

Comment:
```suggestion
    Avoid touching the database directly with PHP library functions. Use the $wpdb object and associated functions instead.
```
The indentation should always be multiples of 4, so this should either have 8 spaces or 4 spaces indentation, not 6.